src/Entity/OgMembership.php

```diff
@@ -255,7 +255,8 @@ public static function baseFieldDefinitions(EntityTypeInterface $entity_type) {
     $fields['uid'] = BaseFieldDefinition::create('entity_reference')
       ->setLabel(t('Member entity ID'))
       ->setDescription(t('The entity ID of the member.'))
-      ->setTargetEntityTypeId('user');
+      ->setSetting('target_type', 'user')
+      ->setDefaultValue(0);
```

Comment:
This could lead to a security issue (which we had in OG7 days), as we need to make sure an OgMembership could not be created for an admin user.
https://github.com/Gizra/og/blob/7.x-2.x/includes/og.membership.inc#L23